packages/generator-volto/README.md

```diff
@@ -11,6 +11,13 @@ npm install -g yo
 npm install -g @plone/generator-volto
 ```
 
+## Compatibility
+
+|Version   |Volto version |
```

Comment:
FYI, Markdown does not care about extra spaces in a table. I tend to avoid using extra spaces because it is maintenance work, but I also understand that extra spaces makes it prettier to the author.

```suggestion
| Plone version | Volto version |
```